parent/pom.xml

```diff
@@ -1237,6 +1237,40 @@
                 <artifactId>elasticsearch-maven-plugin</artifactId>
                 <version>${version.elasticsearch.plugin}</version>
              </plugin>
+             <plugin>
+                <groupId>org.apache.karaf.tooling</groupId>
+                <artifactId>karaf-maven-plugin</artifactId>
+                <version>${version.karaf-maven-plugin}</version>
+                <configuration>
+                   <descriptors>
+                      <descriptor>file://${project.build.directory}/classes/features.xml</descriptor>
+                      <!-- ideally, those profiles should be referenced in the features file -->
+                      <descriptor>mvn:org.apache.karaf.features/framework/${version.karaf}/xml/features</descriptor>
+                      <descriptor>mvn:org.apache.karaf.features/enterprise/${version.karaf}/xml/features</descriptor>
+                   </descriptors>
+                   <distribution>org.apache.karaf.features:framework</distribution>
+                   <javase>1.8</javase>
+                   <framework>
+                      <feature>framework</feature>
+                   </framework>
+                </configuration>
+                <executions>
+                   <execution>
+                      <id>validate</id>
+                      <phase>process-resources</phase>
+                      <goals>
+                         <goal>verify</goal>
```

Comment:
yes, this is just default configuration for  the plugin to avoid customizing in every single place where it's used.